src/Analysis/Ast/Impl/Types/PythonClassType.cs

```diff
@@ -30,7 +31,7 @@
 
 namespace Microsoft.Python.Analysis.Types {
     [DebuggerDisplay("Class {Name}")]
-    internal partial class PythonClassType : PythonType, IPythonClassType, IGenericType, IEquatable<IPythonClassType> {
```

Comment:
Did we decide to make class types not generic anymore?

Reply:
Yeah, I don't think this should be removed. Probably just moved to `PythonClassType.Generics.cs` where this interface is being implemented.

Reply:
Because `IPythonClassType : IPythonClassMember, IGenericType` and R# shows it as unnecessary